src/client.ts

```diff
@@ -351,8 +351,10 @@ export class VocdoniSDKClient {
       throw Error('No election set');
     }
 
-    return ElectionAPI.info(this.url, electionId ?? this.electionId)
-      .then((electionInfo) =>
+    const electionInfo = await ElectionAPI.info(this.url, electionId ?? this.electionId);
```

Comment:
This could be converted to a series of Promises and remove also the `async` from the definition of the function. Not a must but a good practice